<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>热像图</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
    <item>
     <widget class="QLabel" name="lbPicture">
      <property name="minimumSize">
       <size>
        <width>384</width>
        <height>288</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>384</width>
        <height>288</height>
       </size>
      </property>
      <property name="text">
       <string>Picture</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,10,1">
      <item>
       <widget class="QPushButton" name="btnImage">
        <property name="text">
         <string>拍照</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVideo">
        <property name="text">
         <string>录像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>校准</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>保存路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePath">
          <property name="text">
           <string>./graph</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
